website/www/site/layouts/shortcodes/list.html

```diff
@@ -0,0 +1,13 @@
+{{/*
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+ */}}
+ 
+ <span class="{{ .Get `class` }}">{{ .Inner | markdownify }}</span>
```

Comment:
This shortcode shouldn't be named list. Actually isn't it the same as `paragraph` except here we use span? Maybe we could try to make `paragraph` more generic and pass `span` as an argument?